src/Controllers/ProducttypesController.php

```diff
@@ -54,7 +54,7 @@ public function searchAction(): void
                 $this->request->getPost()
             );
 
-            $this->persistent->searchParams = $query->getParams();
+            $this->persistent->searchParams = array('di'=>null) + $query->getParams();
```

Comment:
Same here

Reply:
I have also modified this as per your request.
ProductsController also had the same part, so I modified that as well.